docs/__init__.py

```diff
@@ -1,13 +0,0 @@
-"""
-marimo documentation package.
-
-Copied from https://github.com/marimo-team/marimo/blob/f25319284d439e8d809fab421f27e9226cb00a65/docs/__init__.py#L3
-
-Currently mkdocs-marimo doesn't ship the best way to embed notebooks in mkdocs, they use
-this mechanism instead, which we should replace with mkdocs-marimo when they update the
```

Comment:
Did they ever update anything? (guess not given this PR)

Reply:
They did but didn't fix the fundamental problem I had, which is that the code block errored out due to using a different encoding of wasm to the one they do in javascript. My understanding is that the recommendation is to use the export command to do this directly, but I couldn't find a nice way to do it during the mkdocs build process, hence this half-way PR to make sure we're happy with the interface. I'll let @math-fehr and @compor weigh in case I'm in the minority.